### ISSUE TITLE
ask tuning details, like lr

### ISSUE THREAD

**Issue by ninisy** [FreedomIntelligence/LLMZoo#10]
Can you explain the learning rate Settings of Instruction-based Tuning and Conversation-based Tuning? thanks so much!

**Comment by zhjohnchan** [comment]
Hi @ninisy,

Thanks for your attention!

We updated experimental settings in the updated [technical report](https://github.com/FreedomIntelligence/LLMZoo/blob/main/assets/llmzoo.pdf). We'll release the training code soon.

Best,
Zhihong

**Comment by zhjohnchan** [comment]
Hi @ninisy,

Thanks for your attention!

We have uploaded the training code (see [here](https://github.com/FreedomIntelligence/LLMZoo#-training-by-yourself)). Now you can train `Phoenix`. :-)

Best,
Zhihong
